src/AspNet.Security.OAuth.Zalo/ZaloAuthenticationDefaults.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Authentication;
```

Comment:
Please add the header to these source files. Example:

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/542ac858ea1d1be1ca6fab2dacb37d4fd309248f/src/AspNet.Security.OAuth.Amazon/AmazonAuthenticationDefaults.cs#L1-L5